tests/conftest.py

```diff
@@ -1,15 +1,20 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+import os
 import re
+import sys
 from datetime import timedelta
 
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.dirname(__file__))))
```

Comment:
Yes, but the imports might be from you indtalled package and not the uncommited git changes